src/request/iframe/IFrameApi.js

```diff
@@ -0,0 +1,41 @@
+class IFrameApi {
```

Comment:
Might rename the RequestApi to PopupApi becase that's really what it is? And then move doMigrateAccounts to keys to KeyStore as it gets called from IframeApi also?